blocks/core/ff_module/ff_module-form-marking-controls/_ff_module-form-marking-controls-renderer.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+var React = require('react');
+
+var MarkingControls = require('./ff_module-form-marking-controls');
+
+var data = {
+	modifier: '',
+	controls: {
+		buttons: [{
+			text:'Send Reminder',
+			id: 1,
+			key: 1
+		},
+		{
+			text: 'Mark Now',
+			id: 2,
+			key: 2
+		},
+		{
+			text: 'Release Marks',
+			id: 3,
+			key: 3
+		}],
+		checkableList: {
+			modifier: 'stacked',
+			items: [{
+				label: 'Label text',
+				required: true,
+				type: 'radio',
+				id: 'react-radio-id-1',
+				value: 'radio-value-1',
+				name: 'react-radio-list-group',
+				key: 1
+			},
+			{
+				label: 'Label text',
+				required: true,
+				type: 'radio',
+				id: 'react-radio-id-1',
+				value: 'radio-value-1',
```

Comment:
Sorry I'm not sure I understand by working properly? Do you mean that they need to be paired correctly to the radio inputs? If so they already work, the checkable list does that on its own. 👍 

![screen shot 2016-04-12 at 14 52 52](https://cloud.githubusercontent.com/assets/8876724/14462307/4802df84-00be-11e6-9383-d844290fdd07.png)